test/single_file/unary.c.expected

```diff
@@ -0,0 +1,124 @@
+#include <inttypes.h>
+#include <stdlib.h>
+#include <string.h>
+static int __dredd_enabled_mutation(int local_mutation_id) {
+  static int initialized = 0;
+  static uint64_t enabled_bitset[1];
+  if (!initialized) {
+    const char* dredd_environment_variable = getenv("DREDD_ENABLED_MUTATION");
+    if (dredd_environment_variable) {
+      char* temp = malloc(strlen(dredd_environment_variable) + 1);
+      strcpy(temp, dredd_environment_variable);
+      char* token;
+      token = strtok(temp, ",");
+      while(token) {
+        int value = atoi(token);
+        int local_value = value - 0;
+        if (local_value >= 0 && local_value < 56) {
+          enabled_bitset[local_value / 64] |= (1 << (local_value % 64));
+        }
+        token = strtok(NULL, ",");
+      }
+      free(temp);
+    }
+    initialized = 1;
+  }
+  return enabled_bitset[local_mutation_id / 64] & (1 << (local_mutation_id % 64));
+}
+
+static int __dredd_replace_unary_operator_PreInc_int(int* arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return (*arg)++;
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return --(*arg);
+  if (__dredd_enabled_mutation(local_mutation_id + 2)) return (*arg)--;
```

Comment:
As discussed previously, do we want to replace PreInc with post inc, and the same for PreDec and PostDec?

Reply:
Interesting! Because in C++ the ++ and -- operators do not yield l-values, your C++-oriented code didn't do quite what we wanted for C. I have changed things and have decided to be a little more permissive in the case of C since we're less constrained.